style.css

```diff
@@ -0,0 +1,117 @@
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    overflow: auto;
+}
+
+.logo {
+    height: 50%;
+    width: 10%;
+    display: block;
+    border: solid black 1px;
+}
+
+h1 {
+    margin: 0;
+}
+
+#one {
+    border-top: solid black 1px;
+    overflow: auto;
+}
+
+#two {
+    border-top: solid black 1px;
+    overflow: auto;
+}
+
+#three {
+    border-top: solid black 1px;
+}
+
+#four {
+    border-top: solid black 1px;
+}
+
+nav {
+    position: fixed;
+    top: 0;
+    background: paleturquoise;
+    height: 8%;
+    opacity: 0.5;
+}
+
+ul{
+    list-style-type: none;
+    text-align: center;
+    padding: 1% 0% 0% 0%;
+}
+li {
```

Comment:
needs a line break between last css declaration and this one

Reply:
Done

Reply:
Done